docs/src/lib/sets/SparsePolynomialZonotope.md

```diff
@@ -30,6 +30,7 @@ CurrentModule = LazySets.SparsePolynomialZonotopeModule
 ```@docs
 remove_redundant_generators(::SparsePolynomialZonotope)
 uniqueID(::Int)
+merge_id(::AbstractVector{Int}, ::AbstractVector{Int}, ::AbstractMatrix{N}, ::AbstractMatrix{N}) where {N}
```

Comment:
```suggestion
LazySets.SparsePolynomialZonotopeModule.merge_id(::AbstractVector{Int}, ::AbstractVector{Int}, ::AbstractMatrix{N}, ::AbstractMatrix{N}) where {N}
```
I think that should fix the error. The reason is that this method is not exported anymore.